lib/Migration/Version2040Date20210908101001.php

```diff
@@ -94,6 +101,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		]);
 
 		$table->setPrimaryKey(['id']);
+		$table->addUniqueIndex(['token'], 'token_uniq_idx');
```

Comment:
```suggestion
		$table->addUniqueIndex(['token'], 'cal_appt_token_uniq_idx');
```

because I think on some DBs the indexes are not specific to the table but global and `otken_uniq_idx` could potentially lead to a conflict